plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -1479,25 +1487,6 @@ instance uni ~ DefaultUni => ToBuiltinMeaning uni DefaultFun where
             chooseListDenotation
             (runCostingFunThreeArguments . paramChooseList)
 
-    toBuiltinMeaning _ver CaseList =
```

Comment:
I moved these nearer the bottom to keep the buitlins in the same order as the tags (and hence implementation date).